test/dispatch.js

```diff
@@ -0,0 +1,23 @@
+var test = require('tape')
+
+var inu = require('../')
+
+test('Calling dispatch triggers update function with action passed to dispatch.', function (t) {
+  var initialModel = {initial: true}
+  var expectedAction = {type: 'DISPATCHED'}
+  var app = {
+    init: function () {
+      return {model: initialModel}
+    },
+    update: function (model, action) {
+      t.deepEqual(action, expectedAction, 'action passed to update is the action passed to dispatch')
+      t.end()
```

Comment:
can we also listen to the `actions` stream returned from `start(app)` for the action?